README.md

```diff
@@ -1,7 +1,44 @@
 # pyCascadia
 Implementation of GEBCO cookbook remove-restore and other cleaning of topography/bathymetry. Uses `pyGMT`.
 
-## Shape file closing
+## Installation
+Some dependencies of the package are a bit fiddly to install, especially on Windows.
+We recommend to use a conda environment, and install the more difficult dependencies through `conda-forge`,
+by running the commands below in sequence:
+1. Clone this GitHub repository.
+1. In your local copy of the GitHub repository, create a conda environment named "cascadia" with Python 3.9 and install `pip`, `matplotlib`, `pytest`, and `xarray`.
+```
+conda create -n cascadia python=3.9 pip matplotlib pytest xarray
+```
+1. Additionally, install `pygmt`, `gdal` and `rasterio` into the "cascadia" environment via `conda-forge`.
+```
+conda install -n cascadia pygmt gdal rasterio -c conda-forge
+```
+1. Activate the "cascadia" environment
+```
+conda activate cascadia
+```
+1. Install `pyCascadia`
+```
+pip install .
```

Comment:
I actually think this command might install pycascadia's dependencies automatically (since they're described in `setup.py`)